src/server/routes/editor.js

```diff
@@ -48,14 +48,12 @@ const AchievementForm = React.createFactory(
 const router = express.Router();
 
 router.get('/edit', auth.isAuthenticated, (req, res, next) => {
-	const editorPromise = new Editor({id: parseInt(req.user.id, 10)})
+	const editorJSONPromise = new Editor({id: parseInt(req.user.id, 10)})
 		.fetch()
-		.then((editor) =>
-			editor.toJSON()
-		);
+		.then((editor) => editor.toJSON());
 
-	const titlePromise = new TitleUnlock()
-		.where('editor_id', parseInt(req.user.id, 10))
+	const titleJSONPromise = new TitleUnlock()
+		.where(_.snakeCase('editorId'), parseInt(req.user.id, 10))
```

Comment:
I need to put editor_id into a seperate where call otherwise it returns every unlocked title regardless of user. But where does not convert snake names to camelcase, this removes an error in the linter.